resume-input/src/Components/Basics/Basics.jsx

```diff
@@ -1,34 +1,45 @@
-import React from 'react';
-import './basics.scss';
-
+import React from "react";
+import "./basics.scss";
+import Button from "../Button/Index";

```

Comment:
maybe leave a comment about why the button does not work or what issues you are having

Reply:
When I clicked submit, I couldn't retrieve the value, by the time I got that solved, I only had 30minutes left so I switched to styling instead ^^"